network-api/networkapi/settings.py

```diff
@@ -591,3 +591,6 @@
 
 # RSS / ATOM settings
 FEED_LIMIT = env('FEED_LIMIT')
+
+# Support pages with a large number of fields
+DATA_UPLOAD_MAX_NUMBER_FIELDS = int(env('DATA_UPLOAD_MAX_NUMBER_FIELDS'))
```

Comment:
`env` already does format conversion: we unpack the env all the way at the top, where you indicate which type to expect, and which fallback value to use when it's not defined. If you can update this line to just be `env('DATA_UPLOAD_MAX_NUMBER_FIELDS')` with the appropriate specification at the top of settings.py, then we can safely land this (otherwise, the moment we land this, the website will crash if heroku's missing that var)